Makefile

```diff
@@ -30,7 +30,15 @@ lint:  ## Run linting checks on the project source code (isort, flake8, twine ch
 
 .PHONY: test
 test:  ## Run the project unit tests
-	tox
+	tox tests/unit
+
+.PHONY: integration
+integration:  ## Run the project integration tests
+	-docker-compose -f compose/integration.yaml rm -fsv
```

Comment:
I'm not what this line does? I guess it removes stale containers if there's any.

Reply:
yup! to break it down, a bit, the starting `-` tells make to ignore the output of it (e.g. if there is an error, just move on), `docker-compose rm` will remove the containers associated with the `-f compose/integration.yaml` definition, and the other options:
```
    -f, --force   Don't ask to confirm removal
    -s, --stop    Stop the containers, if required, before removing
    -v            Remove any anonymous volumes attached to containers
```